packages/cashier/src/pages/withdrawal/withdraw/withdraw.tsx

```diff
@@ -1,24 +1,19 @@
 import React from 'react';
-import { useStore, observer } from '@deriv/stores';
-import { Real } from '../../../components/cashier-container';
+import { observer } from '@deriv/stores';
 import { useCashierStore } from '../../../stores/useCashierStores';
+import { WithdrawalFiatModule } from '../../../modules/withdrawal-fiat';
 
 const Withdraw = observer(() => {
-    const { client } = useStore();
-    const {
-        verification_code: { payment_withdraw: verification_code },
-    } = client;
     const { general_store, withdraw } = useCashierStore();
     const { setActiveTab } = general_store;
-    const { container, onMountWithdraw: onMount } = withdraw;
+    const { container } = withdraw;
 
     React.useEffect(() => {
         setActiveTab(container);
-        onMount(verification_code);
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    return <Real />;
+    return <WithdrawalFiatModule />;
```

Comment:
Do you think it'll be a good idea to remove `withdraw` altogether and handle this in `withdrawal.tsx`?